operators/operator_sdk/osdk-cli-reference.adoc

```diff
@@ -12,11 +12,20 @@ This guide documents the Operator SDK CLI commands and their syntax:
 $ operator-sdk <command> [<subcommand>] [<argument>] [<flags>]
 ----
 
+include::modules/osdk-cli-reference-alpha.adoc[leveloffset=+1]
 include::modules/osdk-cli-reference-build.adoc[leveloffset=+1]
+include::modules/osdk-cli-reference-bundle.adoc[leveloffset=+1]
+include::modules/osdk-cli-reference-cleanup.adoc[leveloffset=+1]
 include::modules/osdk-cli-reference-completion.adoc[leveloffset=+1]
-include::modules/osdk-cli-reference-print-deps.adoc[leveloffset=+1]
+include::modules/osdk-cli-reference-create.adoc[leveloffset=+1]
 include::modules/osdk-cli-reference-generate.adoc[leveloffset=+1]
+include::modules/osdk-cli-reference-generate-bundle.adoc[leveloffset=+2]
+include::modules/osdk-cli-reference-generate-kustomize.adoc[leveloffset=+2]
+include::modules/osdk-cli-reference-generate-packagemanifests.adoc[leveloffset=+2]
+include::modules/osdk-cli-reference-init.adoc[leveloffset=+1]
 include::modules/osdk-cli-reference-new.adoc[leveloffset=+1]
-include::modules/osdk-cli-reference-add.adoc[leveloffset=+1]
-include::modules/osdk-cli-reference-test.adoc[leveloffset=+1]
```

Comment:
Did you `git rm` these files that aren't referenced here?  Or are they referenced in other files?

Reply:
I haven't removed these files but didn't find any references. Since they do not apply to 4.6 anymore, may be I will do that.